src/reference/interactive/index.pug

```diff
@@ -17,47 +17,49 @@ block reference
     br
     .alert.alert-info
         p.
-            Looking for Interactive 1 docs? Find them #[a(href='/reference/interactive1/index.html') here].
+            For docs about Interactive 1, go to #[a(href='/reference/interactive1/index.html')Interactive 1 docs].
```

Comment:
I still think the question here is more powerful than repeating "Interactive 1" twice.